src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/PackagePropertyPage.xaml

```diff
@@ -103,6 +103,18 @@
     </StringProperty.DataSource>
   </StringProperty>
 
+  <StringProperty Name="PackageReadmeFile"
```

Comment:
I put this right after PackageIcon since that was the order in the NuGetPackageExplorer. I'm open to moving this around if there is a better spot for it.